app/models/shipit/stack.rb

```diff
@@ -80,6 +80,14 @@ def trigger_deploy(until_commit, user, env: nil)
       deploy
     end
 
+    def trigger_continuous_deploy!
+      return unless deployable?
+      if commit = last_deployable_commit
+        return if commit.deployed?
+        trigger_deploy(commit, commit.committer)
```

Comment:
To stay consistent with the existing code. If I want to change `trigger_deploy` to `trigger_deploy!` I'll do it in another PR / commit.